emily/handler/src/api/models/withdrawal/requests.rs

```diff
@@ -139,6 +146,19 @@ impl UpdateWithdrawalsRequestBody {
                     );
                     withdrawals.push((index, Err(error.clone())));
                 }
+                Err(
+                    ref error @ ValidationError::WithdrawalFulfillmentNotConfirmed(
+                        ref status,
+                        request_id,
+                    ),
+                ) => {
+                    tracing::warn!(
+                        request_id,
+                        ?status,
+                        "failed to update withdrawal: withdrawal is not confirmed, but has a fulfillment."
```

Comment:
I think you mean this right
```suggestion
                        "failed to update withdrawal: fulfillment is only allowed for confirmed withdrawals."
```